dali/base/dautils.cpp

```diff
@@ -908,7 +909,14 @@ StringBuffer &CDfsLogicalFileName::makeXPathLName(StringBuffer &lfnNodeName) con
                     c = toupper(*s);
                     // fall through
                 default:
-                    if (isalnum(c))
+                    if (first == s)
```

Comment:
You also seem to be using _ as a separator in place of ::  which increases the possible clashes.